detox/android/detox/src/main/java/com/wix/detox/espresso/scroll/SwipeHelper.kt

```diff
@@ -0,0 +1,103 @@
+package com.wix.detox.espresso.scroll
+
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.*
+import com.wix.detox.espresso.common.annot.*
+import kotlin.math.min
+import kotlin.math.max
+import kotlin.math.sign
+
+typealias Point = Pair<Double, Double>
+
+object SwipeHelper {
+    fun swipeFastInDirection(direction: Int): ViewAction? {
+        return when (direction) {
+            MOTION_DIR_LEFT -> ViewActions.swipeLeft()
+            MOTION_DIR_RIGHT -> ViewActions.swipeRight()
+            MOTION_DIR_UP -> ViewActions.swipeUp()
+            MOTION_DIR_DOWN -> ViewActions.swipeDown()
+            else -> throw RuntimeException("Unsupported swipe direction: $direction")
+        }
+    }
+
+    fun swipeCustomInDirection(direction: Int, fast: Boolean, offset: Double, startPositionX: Double, startPositionY: Double): ViewAction? {
```

Comment:
Please consider keeping the names consistent. Going from `amount` in the external API to `offset` had me confused. In a sense, `startPointX` vs. `startOffsetPercentX` is also not trivial...